docs/bors-ng-setup.txt

```diff
@@ -0,0 +1,93 @@
+exit # not quite a script; some parts are interactive.
+
+# Requires nginx setup from jenkins-setup.txt
+# https://github.com/bors-ng/bors-ng#step-1-register-a-new-github-app
+# Dashboard URL: https://hijinks.mit.edu:4002/
+
+wget -q -O - https://packages.erlang-solutions.com/debian/erlang_solutions.asc | apt-key add -
+echo 'deb https://packages.erlang-solutions.com/debian stretch contrib' >/etc/apt/sources.list.d/erlang-solutions.list
+apt-get update
+apt-get install esl-erlang elixir postgresql
+
+
+# use this command to generate random secrets when called for
+</dev/urandom tr -dc 'a-zA-Z0-9' | head -c64
+
+useradd -m -U borsng
+
+sudo -u postgres createuser -P borsng # generate random password
+sudo -u postgres createdb -O borsng borsng
+sudo -u postgres psql -d borsng <<<"CREATE EXTENSION IF NOT EXISTS citext;"
```

Comment:
Just to confirm -- postgres creates a localhost-only server, right? There's no chance of someone else being able to connect?

(I know we have a password, but as a second layer of precaution.)

Reply:
Yep, default configuration is 

```
#listen_addresses = 'localhost'         # what IP address(es) to listen on;
                                        # comma-separated list of addresses;
                                        # defaults to 'localhost'; use '*' for all
```